### PR TITLE
Added makefile.shade to run nano tests

### PR DESCRIPTION
**Pull request by kichalla**
[Fixes #348] Only run Nano tests for the WinNano-MusicStore node in CI

@pranavkm

### CODE REVIEW

File: makefile.shade [hunk 1]
```diff
@@ -0,0 +1,28 @@
+default BASE_DIR_LOCAL='${Directory.GetCurrentDirectory()}'
+default IgnoreDotnetTestExitCode = '${E("KOREBUILD_IGNORE_DOTNET_TEST_EXIT_CODE") == "true"}'
```
Comment (pranavkm) [inline makefile.shade]: Should be fine to ignore this.

File: makefile.shade [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+default Configuration='${E("Configuration")}'
```
Comment (pranavkm) [inline makefile.shade]: Just declare these locally inside your target.

File: makefile.shade [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        testArgs += " -trait E2ETests=NanoServer";
+        testArgs += " -parallel none";
+
+        try
+        {
+            Dotnet("test" + testArgs, e2eTestsDirectory);
+        }
+        catch
+        {
+            if (!IgnoreDotnetTestExitCode)
```
Comment (pranavkm) [inline makefile.shade]: Removing this should be fine.

**Comment by pranavkm** [comment]
:watch:

**Comment by kichalla** [comment]
:up: :date:

**Comment by pranavkm** [comment]
:shipit:

**Comment by kichalla** [comment]
2fb9e2f567d5f0947f5c8ca213ae4074a0f57de1

**Pull request closed without merging**